config/locales/en.txt

```diff
@@ -3289,10 +3291,10 @@
   runtime_object_not_found: "Sorry, the [type] you tried to display (id #[id]) no longer exists.  Someone has either deleted it or merged it into another."
   runtime_profile_must_define: You must define this location before we can make it your primary location. Any other changes to your profile have been saved.
   runtime_reverify_already_verified: Your account is already verified!  Please log in.
-  runtime_reverify_sent: Another verification email was sent to your email account.
+  runtime_reverify_sent: Another verification email was sent to your email account. If you do not see it in your inbox, check your spam folder. (If it is in your spam folder, please notify your email provider that it is not spam and whitelist mushroomobserver.org. This may prevent future MO emails from ending up in your -- or other users' -- spam folders.)
```

Comment:
Re-submitted this as a new branch ("spam-notice") with PR #214
<https://github.com/MushroomObserver/mushroom-observer/pull/214>.  Deleted
old branch ("verification-check-spam").

On Thu, Dec 24, 2015 at 7:36 AM Nathan Wilson <notifications@github.com>
wrote:

> In config/locales/en.txt
> <https://github.com/MushroomObserver/mushroom-observer/pull/209#discussion_r48418800>
> :
>
> > @@ -3289,10 +3291,10 @@
> >    runtime_object_not_found: "Sorry, the [type] you tried to display (id #[id]) no longer exists.  Someone has either deleted it or merged it into another."
> >    runtime_profile_must_define: You must define this location before we can make it your primary location. Any other changes to your profile have been saved.
> >    runtime_reverify_already_verified: Your account is already verified!  Please log in.
> > -  runtime_reverify_sent: Another verification email was sent to your email account.
> > +  runtime_reverify_sent: Another verification email was sent to your email account. If you do not see it in your inbox, check your spam folder. (If it is in your spam folder, please notify your email provider that it is not spam and whitelist mushroomobserver.org. This may prevent future MO emails from ending up in your -- or other users' -- spam folders.)
>
> This same text is now repeated 7 times in en.txt. I'd rather see a single
> 'spam_warning' message that gets reused.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MushroomObserver/mushroom-observer/pull/209/files#r48418800>
> .
>